### PR TITLE
chore(apps/pg-api): playground migration

### LINKED ISSUE #978: `/apps/playground-api` - Consolidation & Migration
> As part of #580 consolidation efforts. 
> https://github.com/DeFiCh/playground

#### What would you like to be added:

Migration of the entire `DeFiCh/playground` project into `DeFiCh/jellyfish/apps/playground-api`. This project is different from `DeFiCh/jellyfish/packages/playground`.

**Why are we migrating this?**
We need to maintain this project regardless of the slated deprecation. By migrating this project into `DeFiCh/jellyfish`, we can maintain it in one place.

/sig ecosystem
/area apps
/triage accepted

### PR DESCRIPTION
**Pull request by canonbrother**
#### Which issue(s) does this PR fixes?:

Fixes #978

### CODE REVIEW

File: apps/libs/actuator/src/ActuatorModule.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { HealthCheckError, HealthIndicator, HealthIndicatorResult, TerminusModule } from '@nestjs/terminus'
 import { Global, Module } from '@nestjs/common'
 
+export { HealthIndicatorResult, HealthCheckError }
```
Comment (fuxingloh) [inline apps/libs/actuator/src/ActuatorModule.ts:4]: Should separate this into another PR.

File: apps/libs/actuator/src/index.ts [hunk 2]
```diff
@@ -1,3 +1,6 @@
 export * from './ActuatorModule'
 export * from './ActuatorController'
 export * from './BlockchainCppModule'
+export * from './filters/ErrorFilter'
+export * from './filters/GlobalValidationPipe'
+export * from './filters/ResponseInterceptor'
```
Comment (fuxingloh) [inline apps/libs/actuator/src/index.ts:6]: And this.

Also instead of putting this into the `actuator` module, you should create a new module called `libs/filters`?

File: apps/playground-api/src/PlaygroundIndicator.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@nestjs/common'
+import { ProbeIndicator, HealthIndicatorResult } from '@defichain-apps/libs/actuator'
```
Comment (canonbrother) [inline apps/playground-api/src/PlaygroundIndicator.ts]:
```suggestion
import { ProbeIndicator } from '@defichain-apps/libs/actuator'
import { HealthIndicatorResult } from '@nestjs/terminus'
```

File: apps/libs/actuator/src/ActuatorModule.ts (hunk 1 above)
Comment (canonbrother) [reply apps/libs/actuator/src/ActuatorModule.ts]: yea i just removed it since its from @nestjs/terminus

changes:
https://github.com/DeFiCh/jellyfish/pull/1140/files#r817350854

File: packages/testcontainers/src/index.ts [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import { MasterNodeKey as MNK, RegTestFoundationKeys } from '@defichain/jellyfish-network'
 
 export { DockerOptions } from 'dockerode'
+export { waitForCondition } from './utils'
```
[the remark this answers is not in the document]
Comment (canonbrother) [reply packages/testcontainers/src/index.ts:4]: added https://github.com/DeFiCh/jellyfish/pull/1141

File: apps/libs/filters/src/index.ts [hunk 5]
```diff
@@ -0,0 +1,3 @@
+export * from './ErrorFilter'
+export * from './GlobalValidationPipe'
+export * from './ResponseInterceptor'
```
Comment (fuxingloh) [inline apps/libs/filters/src/index.ts:3]: Maybe should split this into another PR too. I think would be better to keep this atomic.

Comment (canonbrother) [reply apps/libs/filters/src/index.ts]: resolved as #1149

**Review by fuxingloh — APPROVED** [review]

**Review by eli-lim — APPROVED** [review]
